docs/gettingStarted/setupGenericScheme.md

```diff
@@ -206,28 +204,28 @@ If you are using:
               "arcVersion": "0.0.1-rc.22" # choose the correct arc version
             }
 
-     3. If the scheme is for an already existing DAO, then edit `<existing-DAO>.json` for correct network. 
-        Add schemes section eg.
+     3. If the scheme is for an already existing DAO, then edit `<existing-DAO>.json` for the correct network. 
+        Add to the schemes section, eg.
 
               "Schemes": {
                 "GenesisScheme": "0xaddress-of-genericScheme-on-this-network"
                 },
               "arcVersion": "0.0.1-rc.22" # choose the correct arc version
 
 
-## How to get your Generic Scheme shown in Alchemy
+## How to get your Generic Scheme showing up in Alchemy
 
-  To enable a User-friendly Interface for GenericScheme, we have created a way for customizing the UI for specific contracts’ proposals.
+To help you get a user-friendly interface for your generic scheme, we have created a way to customize Alchemy's UI for specific generic schemes.
   
-  The customization will happen at the following places and you will have to submit PR to Alchemy repo with details of your scheme.
+The customization has a few pieces, and you will have to submit a PR to the Alchemy repo once you're finished with it.
 
-### Creating Proposal
+### Proposal Creation Interface
 
-  Customize the create proposal popup to present the different functions the scheme can call on the contract. This requires adding the contracts’ ABI and customizing things like the titles of the labels and placeholders.
+Customize the "create proposal" popup to present the different functions the scheme can call on the contract. This requires adding the contracts’ ABI and customizing things like the titles of the labels and placeholders.
```

Comment:
Yes, the generic scheme can call any function of the contract, but we are asking people to show each of those functions in the UI, rather than expecting users to write in the function names themselves as params, right? That's what I was trying to say here.